### PR TITLE
AGDIGGER-61 - Allow to obtain build logs

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Allow to fetch build logs using java client.

Review @matzew @odra @luigizuccarelli

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,4 @@ buildNumber.properties
 # Intellij Idea files
 .idea
 *.iml
-
+**/*/Main.java
```
Comment (matzew) [inline .gitignore]: hu?

File: src/main/java/org/aerogear/digger/services/BuildService.java [hunk 2]
```diff
@@ -1,17 +1,16 @@
 package org.aerogear.digger.services;
 
 import com.offbytwo.jenkins.JenkinsServer;
-import com.offbytwo.jenkins.model.Executable;
-import com.offbytwo.jenkins.model.JobWithDetails;
-import com.offbytwo.jenkins.model.QueueItem;
-import com.offbytwo.jenkins.model.QueueReference;
+import com.offbytwo.jenkins.model.*;
```
Comment (matzew) [inline src/main/java/org/aerogear/digger/services/BuildService.java]: I think I really hate wildcard import :)

**Comment by matzew** [comment]
:+1:

File: src/main/java/org/aerogear/digger/services/BuildService.java (hunk 2 above)
Comment (wtrocki) [reply src/main/java/org/aerogear/digger/services/BuildService.java]: new intelij option I forgot to change - thanks for spotting this!

File: .gitignore (hunk 1 above)
Comment (wtrocki) [reply .gitignore]: Internal integration tests. Going to remove it :]

**Comment by wtrocki** [comment]
Minor formatting change introduced. See separate commit - going to squash this after review.

**Pull request merged**